docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md

```diff
@@ -2158,6 +2158,78 @@ in the charger into the DUT when the battery level sits between 95% and 98%.
     not charging
     ```
 
+## ECR031.001 EC firmware sync in coreboot
+
+**Test description**
+
+This test verifies the automatic EC firmware update / sync feature in coreboot.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `OPERATING_SYSTEM` = Ubuntu 22.04
+
+**Test setup**
+
+1. Proceed with the
+   [Test cases common documentation](#test-cases-common-documentation) section.
+1. Downgrade to the previous firmware version (both EC and coreboot must be
+   downgraded).
```

Comment:
```suggestion
1. Downgrade to the previous firmware version (both EC and coreboot must be
   downgraded) if needed.
```

It really depends on the starting point

Reply:
good point, applied in https://github.com/Dasharo/docs/pull/806/commits/88c8a69111162f98b2c48c6c9592f20b44ea88ce